instrumentation-genai/opentelemetry-instrumentation-openai-v2/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- Added a wrapper for `AsyncCompletions.create` inside `src/opentelemetry/instrumentation/openai_v2/__init__.py` to instrument async chat completions
```

Comment:
is this typical amount of detail we want in the release notes? I was thinking that from an end user viewpoint, the main change is support for `AsyncOpenAI`/`AsyncCompletions`. The various steps to implement this, might be considered too much information, especially if we repeat this pattern for every update to this package. Thoughts?

Reply:
Yes!

Reply:
noted, will adjust